backend/kitschy_api/managers.py

```diff
@@ -13,17 +13,17 @@ def create_user(self, email, password, **extra_fields):
         user.set_password(password)
         user.save()
         return user
-    
+
     def create_superuser(self, email, password, **extra_fields):
-        """"
+        """ "
```

Comment:
Typo?

Reply:
ay dle bai

Reply:
SS of docs/ path
![image](https://github.com/user-attachments/assets/b7491888-4da0-4867-9d56-011bc105f8a2)